sql/src/main/java/io/crate/metadata/sys/SysMetricsTableInfo.java

```diff
@@ -119,27 +76,41 @@
                 .put("99", m.getValueAtPercentile(99.0))
                 .build()
             ))
-            .put(Columns.P25, () -> forFunction(m -> m.getValueAtPercentile(25.0)))
-            .put(Columns.P50, () -> forFunction(m -> m.getValueAtPercentile(50.0)))
-            .put(Columns.P75, () -> forFunction(m -> m.getValueAtPercentile(75.0)))
-            .put(Columns.P90, () -> forFunction(m -> m.getValueAtPercentile(90.0)))
-            .put(Columns.P95, () -> forFunction(m -> m.getValueAtPercentile(95.0)))
-            .put(Columns.P99, () -> forFunction(m -> m.getValueAtPercentile(99.0)))
-            .put(Columns.NODE, () -> forFunction(ignored -> ImmutableMap.builder()
+            .register("percentiles", "25", LONG, () -> forFunction(m -> m.getValueAtPercentile(25.0)))
+            .register("percentiles", "50", LONG, () -> forFunction(m -> m.getValueAtPercentile(50.0)))
+            .register("percentiles", "75", LONG, () -> forFunction(m -> m.getValueAtPercentile(75.0)))
+            .register("percentiles", "90", LONG, () -> forFunction(m -> m.getValueAtPercentile(90.0)))
+            .register("percentiles", "95", LONG, () -> forFunction(m -> m.getValueAtPercentile(95.0)))
+            .register("percentiles", "99", LONG, () -> forFunction(m -> m.getValueAtPercentile(99.0)))
+
+            .register("node", ObjectType.builder()
+                .setInnerType("id", STRING)
+                .setInnerType("name", STRING)
+                .build(), () -> forFunction(ignored -> ImmutableMap.builder()
                 .put("id", localNode.get().getId())
                 .put("name", localNode.get().getName())
                 .build()
             ))
-            .put(Columns.NODE_ID, () -> forFunction(ignored -> localNode.get().getId()))
-            .put(Columns.NODE_NAME, () -> forFunction(ignored -> localNode.get().getName()))
-            .put(Columns.CLASS, () -> forFunction(h -> ImmutableMap.builder()
+            .register("node", "id", STRING, () -> forFunction(ignored -> localNode.get().getId()))
+            .register("node", "name", STRING, () -> forFunction(ignored -> localNode.get().getName()))
+            .register("classification", ObjectType.builder()
+                .setInnerType("type", STRING)
+                .setInnerType("labels", STRING_ARRAY)
+                .build(), () -> forFunction(h -> ImmutableMap.builder()
```

Comment:
since we touch this code, maybe we can get rid of all guava `ImmutableMap` ?

Reply:
ImmutableMap is heavily used everywhere, do we want to get rid of it ?

Reply:
i'd say so, with java standard utils we active same functionality and have more concise map declaration. don't really see why we would keep using guava utils in this case.

Reply:
Sounds good to me.